content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-policies-cli.md

```diff
@@ -0,0 +1,199 @@
+```
+
+Lastly, `policies apply` also accepts the `-d` option which allows you to add a description of your policy. 
+
+```sh
+chainctl policies apply -f enforce-policy.yaml -d "This is a description of my policy"
+```
+
+Adding a description like this can be helpful in cases where you need to differentiate or organize multiple policies that have been applied to the same group.
+
+
+## Inspect existing policies
+
+If you've already configured one or more policies, you can review them by running the `chainctl policies list` command.
+
+```sh
+chainctl policies list
+```
+
+This will return a table with information about all of your active policies, including the policy's unique identification number, its name, and description if one has been added.
+
+```
+                             ID                             |      NAME      |       DESCRIPTION        
+------------------------------------------------------------+----------------+--------------------------
+  d22957f0271ef73221005f8c11bb027888829e09/2eb7b5401728bd9f | sample-policy  | This is a sample policy  
+  d22957f0271ef73221005f8c11bb027888829e09/6c559b7225e889f9 | my-policy      | My policy                
+  d22957f0271ef73221005f8c11bb027888829e09/f7253a47e09f3bf0 | enforce-policy |                          
+```
+
+However, this output only provides some high-level information about your policies. In order to inspect the contents of a given policy, you can run the `policies view` subcommand.
+
+```sh
+chainctl policies view
+```
+
+A menu will appear, prompting you to select the policy you'd like to inspect. To avoid using the interactive menu, you can follow the `view` subcommand with the name or identification number of the policy you want to inspect.
+
+```sh
+chainctl policies view enforce-policy
+```
+
+This will output the contents of the chosen policy directly.
+
+
+## Edit policies
+
+`chainctl` includes the `policies edit` subcommand, allowing you to modify existing policies.
+
+```sh
+chainctl policies edit
+```
+
+As with `policies view`, when you run `policies edit` without any arguments it will open an interactive menu from which you can select the policy that you want to edit. You can also append the name or the identification number Chainguard Enforce uses to refer to the policy to avoid having to select the policy, as in this example.
+
+```sh
+chainctl policies edit enforce-policy
+```
+
+Regardless of how you run this command, it will open your default text editor where you can edit the specified policy's contents. After saving your changes and closing the editor, `chainctl` will prompt you to confirm whether you want to save the changes to the active policy. If you do confirm, the updated policy will be saved and applied automatically.
+
+Be aware that if you used a `.yaml` file as the basis for the policy you applied to your cluster, the `edit` subcommand will not edit the original file. It will only edit the policy as it exists within Chainguard Enforce.
```

Comment:
Maybe make this a note, like 

```
> **Note:**
```

This also seems kind of counterintuitive. Do we have a little more guidance on why this is?

Reply:
I tried to add some clarification, but please let me know if you think it needs more explanation.